### PR TITLE
add key shorcut to skip dialogue

### PR DESCRIPTION
**Pull request by napstaa967**
saw what kade said on my last pr so i turned the option into a key shorcut

if you enable the skip button on gameplay options and press escape/backspace you can skip cutscenes

### CODE REVIEW

**Comment by napstaa967** [comment]
i am trying to test in thorns but the game crashes as soon as i load in the song

**Comment by Lucky-56** [comment]
then mark this as a draft

**Comment by napstaa967** [comment]
ok now that funny spirit is fixed i have to test it

**Comment by napstaa967** [comment]
i can't get thorns to work it ignores all inputs

**Review by raf13lol — APPROVED** [review]
looks good to go

**Review by Lucky-56 — CHANGES_REQUESTED** [review]
you forgot to set a default value for `cutscenes` in `KadeEngineData.hx`, else it looks good.

**Comment by napstaa967** [comment]
> you forgot to set a default value for `cutscenes` in `KadeEngineData.hx`, else it looks good.

i forgot

File: source/KadeEngineData.hx [hunk 1]
```diff
@@ -14,6 +14,9 @@ class KadeEngineData
 
 		if (FlxG.save.data.downscroll == null)
 			FlxG.save.data.downscroll = false;
+	    
+	    	if (FlxG.save.data.cutscenes == null)
+			FlxG.save.data.downscroll = false;
```
Comment (Lucky-56) [inline source/KadeEngineData.hx]: heh, I hope you see what's wrong...
if not, `FlxG.save.data.downscroll` should be `FlxG.save.data.cutscenes`.

**Review by Lucky-56 — CHANGES_REQUESTED** [review]

**Review by Lucky-56 — APPROVED** [review]
looks good

**Comment by Kade-github** [comment]
Again I generally just don't want this to be an option. it would clutter shit up.

**Pull request closed without merging**